includes/reader-activation/class-reader-activation.php

```diff
@@ -1699,6 +1690,96 @@ public static function register_reader( $email, $display_name = '', $authenticat
 		return $user_id;
 	}
 
+	/**
+	 * Get user data args for creating a new WP user.
+	 * See https://developer.wordpress.org/reference/functions/wp_insert_user/ for supported args.
+	 *
+	 * @param array $user_data          Default args for the new user.
+	 *              $user_data['email] Email address for the new user (required).
+	 */
+	public static function get_user_data( $user_data = [] ) {
```

Comment:
NIT: Can we rename this to `sanitize_user_data()`?

Reply:
NIT: NIT:  I'd rather opt for ["canonize"](https://en.wikipedia.org/wiki/Canonicalization), since data is added, not only sanitized

Reply:
Renamed to `canonize_user_data` in 6e3f10c8